src/ETISS.cpp

```diff
@@ -776,15 +796,195 @@ void etiss_initialize(std::vector<std::string> args, bool forced = false)
+                                           "functions. Please add the line \'etiss::initialized(argc,argv);\' "
+                                           "at the beginning of \'int main(int argc,char**argv);\'");
+            }
+        }
+        initialized_ = true;
+    }
+    
+    {
+        namespace po = boost::program_options;
+        try 
+        {
+            po::options_description desc("Allowed options");
+            desc.add_options()
+            ("help", "produce a help message")
+            /*("norangeexception", po::value<std::string>(), "just an option (flag)")
+            ("returnjump", po::value<std::string>(), "just an option (flag)")
+            ("hostendianness", po::value<std::string>(), "just an option (flag)")
+            ("ignore_sr_iee", po::value<std::string>(), "just an option (flag)")
+            ("cleanup", po::value<std::string>(), "just an option (flag)")
+            ("verifyJIT", po::value<std::string>(), "just an option (flag)")
+            ("copy-headers", po::value<std::string>(), "just an option (flag)")
+            ("sigint-console", po::value<std::string>(), "just an option (flag)")
+            ("library-loading", po::value<std::string>(), "just an option (flag)")
+            ("integrated-library", po::value<std::string>(), "just an option (flag)")
+            ("pyconsole", po::value<std::string>(), "just an option (flag)")
+            ("jit-debug", po::value<std::string>(), "just an option (flag)")
+            ("ETISS::enable_dmi", po::value<std::string>(), "just an option (flag)")
+            ("ETISS::log_pc", po::value<std::string>(), "just an option (flag)") */
+            ("logaddr", po::value<std::string>(), "Plugin Logger option")
+            ("logmask", po::value<std::string>(), "Plugin Logger option")
+            ;
+
+            po::command_line_parser parser{argc, argv};
+            po::command_line_parser iniparser{argc, argv};
+            iniparser.options(desc).allow_unregistered().extra_parser(inifileload).run();
+            parser.options(desc).allow_unregistered().extra_parser(etiss::Configuration::set_cmd_line_boost);
+            po::parsed_options parsed_options = parser.run();
+            po::store(parsed_options, vm);
+
+            if (vm.count("help")) {
+                std::cout << desc;
+                std::cout << "\nPlease begin all options with --\n";
+            }
+
+            auto unregistered = po::collect_unrecognized(parsed_options.options, po::include_positional);
+            for (auto iter_unreg : unregistered)
+            {
+                if (iter_unreg.find("-i") != 0 )
+                {
+                    std::cout << "\nUnrecognised option "<< iter_unreg<< ".\n";
+                    std::cout << "Please use --help to list all recognised options. \n\n";
+                }
+            }
+        }
+    catch(std::exception& e) {
```

Comment:
To fix all indentation and formatting issues automatically, you can use the "clang-format" tool